code/modules/admin/sql_ban_system.dm

```diff
@@ -411,6 +411,8 @@
 	reason = href_list["reason"]
 	if(!reason)
 		error_state += "No reason was provided."
+	if(length(reason) > 600)
```

Comment:
Could you turn this into a define, instead of a magic number?

One that is shared by same sources of limitation, if possible.